server/libs/infra/src/repositories/asset.repository.ts

```diff
@@ -214,6 +214,9 @@ export class AssetRepository implements IAssetRepository {
 
   async getMapMarkers(ownerId: string, options: MapMarkerSearchOptions = {}): Promise<MapMarker[]> {
     const { isFavorite, fileCreatedAfter, fileCreatedBefore } = options;
+    const coordinateFilter = Raw(
```

Comment:
We should consider removing this code in the future. This is really only a fix for old, previously incorrectly parsed data. Alternatively we could recommend users running the "all" exif job to fix it. Or, we could include a migration in the next release which would update these values to `null`.

Reply:
Good point, will make a PR